docs/source/changes/260.add_remote_drone_draining.yaml

```
@@ -0,0 +1,8 @@
+category: added
+summary: "Added support for remote draining of drones i.e. using REST API"
```

I think this makes it clearer that this is something the admin does.
```suggestion
summary: "Added support for manual draining of drones using the REST API"
```